src/mscorlib/shared/System/Text/StringBuilder.cs

```diff
@@ -1204,55 +1158,15 @@ public StringBuilder Insert(int index, String value)
             return this;
         }
 
-        // Returns a reference to the StringBuilder with ; value inserted into 
-        // the buffer at index. Existing characters are shifted to make room for the new text.
-        // The capacity is adjusted as needed. If value equals String.Empty, the
-        // StringBuilder is not changed.
-        // 
-        public StringBuilder Insert(int index, bool value)
-        {
-            Contract.Ensures(Contract.Result<StringBuilder>() != null);
-            return Insert(index, value.ToString(), 1);
-        }
+        public StringBuilder Insert(int index, bool value) => Insert(index, value.ToString(), 1);
 
-        // Returns a reference to the StringBuilder with ; value inserted into 
-        // the buffer at index. Existing characters are shifted to make room for the new text.
-        // The capacity is adjusted as needed. If value equals String.Empty, the
-        // StringBuilder is not changed.
-        // 
         [CLSCompliant(false)]
-        public StringBuilder Insert(int index, sbyte value)
-        {
-            Contract.Ensures(Contract.Result<StringBuilder>() != null);
-            return Insert(index, value.ToString(), 1);
-        }
+        public StringBuilder Insert(int index, sbyte value) => Insert(index, value.ToString(), 1);
 
-        // Returns a reference to the StringBuilder with ; value inserted into 
-        // the buffer at index. Existing characters are shifted to make room for the new text.
-        // The capacity is adjusted as needed. If value equals String.Empty, the
-        // StringBuilder is not changed.
-        // 
-        public StringBuilder Insert(int index, byte value)
-        {
-            Contract.Ensures(Contract.Result<StringBuilder>() != null);
-            return Insert(index, value.ToString(), 1);
-        }
+        public StringBuilder Insert(int index, byte value) => Insert(index, value.ToString(), 1);
 
-        // Returns a reference to the StringBuilder with ; value inserted into 
-        // the buffer at index. Existing characters are shifted to make room for the new text.
-        // The capacity is adjusted as needed. If value equals String.Empty, the
-        // StringBuilder is not changed.
-        // 
-        public StringBuilder Insert(int index, short value)
-        {
-            Contract.Ensures(Contract.Result<StringBuilder>() != null);
-            return Insert(index, value.ToString(), 1);
-        }
+        public StringBuilder Insert(int index, short value) => Insert(index, value.ToString(), 1);
 
-        // Returns a reference to the StringBuilder with ; value inserted into 
```

Comment:
@AlexGhiondea I believe this is fairly obvious from the naming of the method. The comment only takes up a lot of space.